File: helidon/src/main/java/com/github/panga/app/Product.java
package com.github.panga.app;

public class Product {

	private final String id;
	private final String name;
	private final Float price;
	private final Integer weight;

	public Product(final String id, final String name, final Float price, final Integer weight) {
		this.id = id;
		this.name = name;
		this.price = price;
		this.weight = weight;
	}

	public String getId() {
		return id;
	}

	public String getName() {
		return name;
	}

	public Float getPrice() {
		return price;
	}

	public Integer getWeight() {
		return weight;
	}
}

File: swift_vapor/Sources/App/Models/Product.swift
import Vapor

final class Product: Content {
    var id: String
    var name: String
    var price: Double
    var weight: Int

    init(id: String, name: String, price: Double, weight: Int) {
        self.id = id
        self.name = name
        self.price = price
        self.weight = weight
    }
}
